platform/security/sts/security-sts-propertyclaimshandler/src/main/java/org/codice/ddf/security/sts/claims/property/PropertyFileClaimsHandler.java

```diff
@@ -160,7 +169,30 @@ public void setPropertyFileLocation(String propertyFileLocation) {
     if (propertyFileLocation != null
         && !propertyFileLocation.isEmpty()
         && !propertyFileLocation.equals(this.propertyFileLocation)) {
-      userMapping = PropertiesLoader.toMap(PropertiesLoader.loadProperties(propertyFileLocation));
+
+      Properties p = new Properties();
+      try {
+        p.load(getClass().getResourceAsStream(propertyFileLocation));
+      } catch (FileNotFoundException e) {
+        LOGGER.debug("File not found when attempting to load user properties file.", e);
+      } catch (IOException e) {
+        LOGGER.debug("Exception when trying to load the user properties file.", e);
+      }
```

Comment:
❗️ I believe you should use `PropertiesLoader.loadProperties(propertyFileLocation)` to get the `Properties` object instead of doing it manually here with this try/catch block. That will inherit built-in cross-OS compatibility and handling for relative versus absolute paths.

Reply:
I can't see an easy way to convert from `java.util.Properties` (produced by `PropertiesLoader.loadProperties`) to `org.apache.felix.utils.properties.Properties` required by `PropertiesBackingEngine`.

Reply:
I've reworked the files to include a static variable for the users.properties file so the only time the setter is used will be via a test. Also removed the users.property file entry from the blueprint